scripts/sct_utils.py

```diff
@@ -391,6 +391,25 @@ def tmp_create(verbose=1):
     return path_tmp
 
 
+def delete_tmp_files_and_folders(path='', verbose=1):
```

Comment:
This is better. You can move the imports to the top of the file and remove the verbose parameter (it is not used in the function)